riscv/src/compiler.rs

```diff
@@ -741,6 +767,27 @@ fn preamble() -> String {
         Z = X_b1 + X_b2 * 0x100 + X_b3 * 0x10000 + X_b4 * 0x1000000
     }
 
+    // implements Z = Y % X, stores quotient in `div_tmp`.
+    instr remu Y, X -> Z {
```

Comment:
after the `multi-return` PR we can probably merge this and the `divu` instructions and return both at the same time

Reply:
let me write a TODO in the code